### ISSUE TITLE
Project will not run due to error with missing value for ADDIN_DOMAIN

### ISSUE THREAD

**Issue by AE-MS** [OfficeDev/extendteamsapp#12]
If I clone the project, open in VS Code, and hit F5, it refuses to run with this error:

```
  (×) Error: teamsApp/zipAppPackage failed.
    (×) Error: The program cannot proceed as the following environment variables are missing: 'ADDIN_DOMAIN', which are required for file: d:\repos\extendteamsapp\appPackage\manifest.json. Make sure the required variables are set either by editing the .env file 'd:\repos\extendteamsapp\env\.env.local' with the correct names and values , or by setting the system environment variables with the correct names and values. If you are developing with a new project created with Teams Toolkit, running provision or debug will register correct values for these environment variables.
```

I didn't see any obvious instructions in the README regarding how this should be set.

**Comment by sbtron** [comment]
Temporarily adding documentation for needed env changes while we look for a better solution that just works when cloned -  #15

**Comment by sbtron** [comment]
Fixed by #17
